src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2429,13 +2429,13 @@ export class StudyViewPageStore {
     }
 
     @action.bound
-    addGenomicProfilesFilter(profiles: string[][]): void {
+    addGenomicProfilesFilter(chartMeta: ChartMeta, profiles: string[][]): void {
```

Comment:
it's weird that typescript wouldn't have caught this.  we should figure out why it didn't @Luke-Sikina.  i mena, wherever this was called ought to have broken if it tried to pass ChartMeta to string[][]